.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "ecmaFeatures": {
+    "jsx": true
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "quotes": 1,
+    "no-trailing-spaces": 1,
+    "strict": 1,
```

Comment:
According to http://eslint.org/docs/rules/strict.html this is deprecated. 
I've never fully understood 'use strict', what's the argument for it?